src/core/lua/luacfb.cpp

```diff
@@ -19,15 +19,14 @@
 #include "criticalregion.h"
 
 CLuaCFB::CLuaCFB(CStringDictionary::TStringId paInstanceNameId, const CLuaCFBTypeEntry* paTypeEntry, SCFB_FBNData &paFbnData, forte::core::CFBContainer &paContainer) :
-    CGenFunctionBlock<CCompositeFB>(paContainer, paTypeEntry->getInterfaceSpec(), paInstanceNameId, paFbnData), mTypeEntry(paTypeEntry) {
+    CGenFunctionBlock<CCompositeFB>(paContainer, paInstanceNameId, paFbnData), mTypeEntry(paTypeEntry) {
 }
 
 CLuaCFB::~CLuaCFB() = default;
 
 
 bool CLuaCFB::initialize() {
   //before calling super we need to configure the interface of the FB
-  setupFBInterface(getFBInterfaceSpec());
   return CGenFunctionBlock<CCompositeFB>::initialize();
 }
```

Comment:
I would suggest to remove this method, as well.

Reply:
done